### PR TITLE
Update cell table description to include morphology metrics

### LINKED ISSUE #393: Add cell table description to documentation
**Is your feature request related to a problem? Please describe.**

Now that the cell table contains more morphology metrics (those derived from automatically-calculated regionprops values), it's best that we describe what these new metrics are and how they are computed.

**Describe the solution you'd like**

We'll add this description in the Data Type Info page under cell table.

### LINKED ISSUE #398: Add documentation for morphology metrics
**Is your feature request related to a problem? Please describe.**
We should add documentation explaining the key metrics we've added to the repo

### PR DESCRIPTION
**Pull request by alex-l-kong**
**What is the purpose of this PR?**

Closes #393 and closes #398. A lot of morphology metrics have been added to the cell table, so we should explain what each of the metrics are computing.

**How did you implement your changes**

We add the description to `data_types.md`.

### CODE REVIEW

File: docs/_rtd/data_types.md [hunk 1]
```diff
@@ -21,7 +21,27 @@ Used by: [calculate_channel_spatial_enrichment](https://ark-analysis.readthedocs
...
 
-Description: This holds the extracted single cell data. Each row is a unique cell, and each column is a feature for that cell. This includes the counts for each marker, morphological information such as area and diameter, and information to link that cell back to the original image such as segmentation id and FOV.
+Description: This holds the extracted single cell data. Each row is a unique cell, and each column is a feature for that cell. This includes the counts for each marker, morphological information, and information to link that cell back to the original image such as segmentation id and FOV.  
+
+For each cell, these are the specific morphology metrics computed:
+
+* `cell_size`: the signal intensity. May not be the same as area depending on the signal extraction method used.
+* `area`
+* `eccentricity`
+* `major_axis_length`
```
Comment (ngreenwald) [inline docs/_rtd/data_types.md:31]: You can fill in the rest of these from skimage

File: docs/_rtd/data_types.md [hunk 2]
```diff
@@ -21,7 +21,27 @@ Used by: [calculate_channel_spatial_enrichment](https://ark-analysis.readthedocs
...
+* `minor_axis_length`
+* `perimeter`
+* `convex_area`: the area of the convex hull
+* `equivalent_diameter`: the diameter of the circle with the same area as the cell
+* `centroid-0`: the x-coordinate of the centroid
+* `centroid-1`: the y-coordinate of the centroid
+* `major_minor_axis_ratio`: the major axis length divided by the minor axis length
+* `perim_square_over_area`: the square of the perimeter divided by the area
```
Comment (ngreenwald) [inline docs/_rtd/data_types.md:39]: I think some explanation of why we have these would be useful. You can check out the preprint for a summary of the main ones

**Review by ngreenwald — CHANGES_REQUESTED** [review]
Nice start. We should add a link to the segmentation notebook that links to this specific subsection so people know where to look

File: docs/_rtd/data_types.md [hunk 3]
```diff
@@ -21,7 +21,29 @@ Used by: [calculate_channel_spatial_enrichment](https://ark-analysis.readthedocs
 [create_neighborhood_matrix](https://ark-analysis.readthedocs.io/en/latest/_markdown/ark.segmentation.html#ark.segmentation.marker_quantification.generate_cell_data)  
 Shape: 2D matrix of cells x cell_features  
 
-Description: This holds the extracted single cell data. Each row is a unique cell, and each column is a feature for that cell. This includes the counts for each marker, morphological information such as area and diameter, and information to link that cell back to the original image such as segmentation id and FOV.
+Description: This holds the extracted single cell data. Each row is a unique cell, and each column is a feature for that cell. This includes the counts for each marker, morphological information, and information to link that cell back to the original image such as segmentation id and FOV.  
+
+For each cell, the following morphology features calculated from `skimage.measure.regionprops` are included:
+
+* `cell_size`: the signal intensity. May not be the same as area depending on the signal extraction method used.
```
Comment (ngreenwald) [inline docs/_rtd/data_types.md:28]: I think cell size is just the size, can you confirm?

**Review by ngreenwald — CHANGES_REQUESTED** [review]
We should add a comment to the notebook in the cell that creates the cell table with a link to this section of the documentation directing people to go there if they have questions about the output data

File: docs/_rtd/data_types.md (hunk 3 above)
Comment (alex-l-kong) [reply docs/_rtd/data_types.md:28]: It is the value of channel counts as returned by `signal_extraction.py`. We should probably rename it to something more descriptive, such as `channel_counts` or `signal_intensity`.

Comment (ngreenwald) [reply docs/_rtd/data_types.md:28]: Channel counts is the count for each channel. How could that be the value in a single column in the cell table?
This is where cell_size is defined: https://github.com/angelolab/ark-analysis/blob/db9065d18aab74aad0bfea6e275eede661b7a85a/ark/segmentation/marker_quantification.py#L119

Comment (alex-l-kong) [reply docs/_rtd/data_types.md:28]: @ngreenwald OK, I got confused between cell size and cell (channel) counts.

**Review by ngreenwald — APPROVED** [review]

**Pull request merged**